modules/system/lang/pt-br/validation.php

```diff
@@ -100,7 +129,11 @@
     |
     */
 
-    'custom' => [],
+    'custom' => [
+        'attribute-name' => [
+            'rule-name' => 'custom-message',
+        ],
+    ],
```

Comment:
```suggestion
    'custom' => [],
```
this is an example, not an actual message to be translated.

Reply:
had copied from the original English version, just to make it the same.